src/Workleap.DomainEventPropagation.Subscription.PullDelivery/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,24 @@
+#nullable enable
```

Comment:
Ça va dans Shipped.txt!

Reply:
See [this reply](https://github.com/gsoft-inc/wl-domain-event-propagation/pull/101/files#r1505120999)